quickstarts/dotnet/blazor/config.yml

```diff
@@ -0,0 +1,52 @@
+id: 
+slug: blazor
+description: |
+  ## Comprehensive monitoring quickstart for Blazor
+
+  The majority of realistic application UI scenarios are optimized for high performance using Blazor. The optimal performance, however, depends on developers implementing the right patterns and features. Use your application rendition's dashboard to visually track the web user experience worldwide.
+
+  ## Why monitor Blazor?
+
+  - Because Blazor uses WebAssembly, which is a feature of all popular browsers, we can utilise Blazor to execute our programme in any browser, including mobile ones. Unlike the days of Flash and Silverlight, we are no longer reliant on plugins. Additionally, Blazor enables us to utilise our C# expertise
+  - Since C# is a fantastic programming language and is tightly typed, we can catch errors at build time rather than during runtime. Blazor is merely a framework that runs on the.NET runtime, therefore we are free to use whatever library we like as long as it is.NET Standard compliant
+  - This means that practically all publicly available NuGet packages as well as our own libraries can be used
+  - There are a few compelling arguments that support the employment of Blazor
+    1. All of the major browsers support WebAssembly
+    2. For interactive web apps, use C#
+    3. Utilize current libraries
+    4. Near-native performance
+    5. Debugging and tooling
+  
+  ## What’s included in this quickstart?
+
+  New Relic's Blazor monitoring quickstart boasts instant full-stack observability out-of-the-box:
+  - Alerts
+  - Dashboards (Throughput, Error rate, Logs, Web transaction Time, Apdex Score, and more)
+  - Monitor scripts and functions
+  - Monitor web transactions
+  
+summary: |
+  Get a comprehensive understanding of performance with customized dashboards including Error rate, response time, web transactions, and more with New Relic’s Blazor monitoring tool
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Blazor
+documentation:
+  - name: Blazor installation docs
+    description: |
+      Software framework for building an object-oriented layer for tracking
+      business logic.
+    url: >-
+      https://docs.newrelic.com/docs/agents/net-agent/getting-started/compatibility-requirements-net-framework-agent
+keywords:
+  - apm
+  - dotnet
```

Comment:
It could be worth adding `.net` as a keyword as well, that way it's covered no matter how a user searches for it!

Reply:
done 👍